datajunction-server/datajunction_server/models/history.py

```diff
@@ -47,6 +47,7 @@ class EntityType(str, Enum):
     NODE = "node"
     QUERY = "query"
     TAG = "tag"
+    DISPLAY_NAME = "display_name"
```

Comment:
Hmm, I think we don't need a specific 'display_name' entity in history. The entity should be `column` and the action `update`